### DISCUSSION TITLE
Histogram of the product of two observables

### DISCUSSION THREAD

**Discussion by jasper7427** [MadAnalysis/madanalysis5#270] [Q&A]
Hi, 
I would like the plot the product of the rapidities of two muons or antimuons in my events. I haven't been able to figure out how to do this.
Thanks

**Comment by BFuks** [comment]
Hi Jasper,

This is in the manual and in the documentation shipped with the code... 
```
define mmu = mu+ mu-
plot eta(mmu[1])
plot eta(mmu[2])
```

If this is not sufficient, consider using the expert mode of the code.

Cheers,

Benjamin

**Reply by jasper7427** [reply]
Hi Benjamin, 

Thanks for your answer. 

I am looking for something that essentially would do 

`plot ETA(mu+[1]) * ETA(mu+[1])`

Is this not possible in normal mode?

Thanks

**Comment by BFuks** [comment]
In fact not.

Regards,

Benjamin
